README.md

```diff
@@ -84,136 +85,88 @@ An agent component for Java system.
-```
-observability.outputServer.bootstrapServer = [ip:port]
-```
-
-* Modify output configuration, if you want to watch log information in console.
-```
-# metric output
-observability.metrics.[xxxx].appendType=console
-
-# tracings output to console
-observability.tracings.output.target=system
-observability.tracings.output.enabled=false
-```
-If you want to watch tracing information only in console, comment out the outputServer configuration and set target to 'system':
-```
-# observability.outputServer.bootstrapServer=127.0.0.1:9092
-# observability.outputServer.timeout=10000
-# observability.outputServer.enabled=true
-
-observability.tracings.output.target=system
-
-```
-
-* Sending tracing data to zipkin server
-```
-# [zipkin]: send data to zipkin server
-# [system]: send data to kafka
-observability.tracings.output.target=zipkin
-observability.tracings.output.target.zipkinUrl=http://localhost:9411/api/v2/spans
-```
+By default, there is an agent.properties configuration , which is configured to print all information to the console.
 
-### 3. Building the demo application and simple plugin.
-```
-$ git clone https://github.com/megaease/easeagent-test-demo.git
-$ cd spring-web
-$ mvn clean package
+### Monitor Spring Petclinic
 
-$ cd simple-plugin
-$ mvn clean package
-```
-There is an agent.properties configuration file in the demo directory, which is configured to print all information to the console.
-If you want to print all information to console, then you can use this configuration file.
+Download and init project then start it. You need git and docker:
 
-### 4. Copy simple plugin to `plugins` directory
-There is a [simple plugin](https://github.com/megaease/easeagent-test-demo/tree/master/simple-plugin) which only for demonstrating the use of the plugin. 
-The compiled simple plugin JAR package should be copy to the `plugins` directory located in `EASE_AGENT_PATH`, and if the directory is not exsist, it need to be created.
+[Project Details](https://github.com/megaease/easeagent-spring-petclinic)
 
 ```
-$ export EASE_AGENT_PATH=[Replace with agent path]
-cp target/simple-plugin-1.0.jar $EASE_AGENT_PATH/plugins/
-
+$ git clone https://github.com/megaease/easeagent-spring-petclinic.git
+$ cd easeagent-spring-petclinic
+$ git checkout v2.1.0
+$ git submodule update --init
+$ ./spring-petclinic.sh start
 ```
 
-### 5. Run the demo application with EaseAgent.
-```
-# Open another console
-$ export EASE_AGENT_PATH=[Replace with agent path]
-$ java "-javaagent:${EASE_AGENT_PATH}/easeagent-dep.jar" -Deaseagent.config.path=${EASE_AGENT_PATH}/agent.properties -Deaseagent.server.port=9900 -jar target/spring-web-1.0.jar
+> The script will download the easeagent v2.1.0 release from the Github release page. 
+> If you want to use your own built EaseAgent, copy it to directory: `easeagent/downloaded`
+>> ```$ cp $EASE_AGENT_PATH/easeagent.jar  easeagent/downloaded/easeagent-v2.1.0.jar``` 
```

Comment:
How about to download the latest version instead of  a specific version. 

The follow document could help on this.
https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases

Reply:
The repository of `easeagent-spring-petclinic`  has a tagged named v2.1.0, which aligns to the same version of the `easeagent` . We could refactor the scripts in the `main branch` to use the latest version of easeagent.

Reply:
> How about to download the latest version instead of a specific version.
> 
> The follow link could help on this. https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8
> 
> Or, take a look the Easegress' installation scripts: https://github.com/megaease/easegress/blob/dce1b84271be2406d0f2ad86912e9b8dbbfc18e7/scripts/install.sh#L30

done